space2stats_api/src/space2stats_ingest/main.py

```diff
@@ -21,12 +21,12 @@ def read_parquet_file(file_path: str) -> pa.Table:
             table = pq.read_table(tmp_file.name)
     else:
         table = pq.read_table(file_path)
-    return table
+    return table.rename_columns([col.lower() for col in table.column_names])
 
 
 def get_stac_fields_from_item(stac_item_path: str) -> Set[str]:
     item = Item.from_file(stac_item_path)
-    columns = [c["name"] for c in item.properties.get("table:columns")]
+    columns = [c["name"].lower() for c in item.properties.get("table:columns")]
```

Comment:
Is there any reason that the STAC Item would have non-lowered table names?  Is this for backwards compatibility?

Reply:
they shouldn't, but they are user generated. We are agreeing on conventions of keeping lowercase for column names. Do you think we shouldn't do this to throw an error that forces re-writing the stac metadata?

Reply:
Hmm... Nah, handling the deprecated format is probably fine also.

Let's go with what you've got!